.github/CONTRIBUTING.md

```diff
@@ -47,15 +55,15 @@ If you want to publish your local changes to this project you have to create a n
 ## Code-Review
 
 After opening the pull-request, your code will be reviewed and if your pull-request passes the review and build server steps, your changes will be merged to master.
```

Comment:
### Review
I think this sentence could be split into shorter, less complicated ones:

```suggestion
After creating the pull-request, your code will be reviewed by other developers. If your pull-request passes the review and the build server accepts your changes, they will be merged into the master branch.
```
This may help to better understand the statement of this sentence when reading it for the first time :)

**Documentation**
The documentation is concise and good to read. The changes made in this PR help to improve the contributing documentation, which is the starting point for new participants of the Elektra community. Therefore, the improvements of this file / this PR does help to reduce entry barriers. 

**Examples**
The examples of the contributing documentation are well chosen and give the necessary insight on how to start to participate for Elektra.

**Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)**
No code changes were made.

**APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)**
No API changes were made.

**Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)**
No code changes were made.